pkg/oc/bootstrap/docker/up.go

```diff
@@ -1011,17 +1016,14 @@ func (c *ClientStartConfig) InstallMetrics(out io.Writer) error {
 // CheckServiceCatalogPrereqVersion ensures the OpenShift server version is high enough to
 // run the service catalog.
 func (c *ClientStartConfig) CheckServiceCatalogPrereqVersion(out io.Writer) error {
-	serverVersion, _ := c.OpenShiftHelper().ServerPrereleaseVersion()
-	// 3.6.0-alpha2 was the last release that did not allow the creation of namespace rolebindings w/o first
-	// creating a policybinding object.  This limitation prevents the service catalog template from instantiating.
-
-	// special case for someone who is building a local image using commits based on 3.6.0.alpha2.  They most likely
-	// have the necessary changes.
-	if serverVersion.EQ(openshiftVersion36alpha2) && (serverVersion.String() != openshiftVersion36alpha2.String()) {
+	if c.ImageVersion == "latest" {
 		return nil
 	}
-	if serverVersion.LTE(openshiftVersion36alpha2) {
-		return errors.NewError("Enabling the service catalog requires a newer server level than %v, this server is version %v", openshiftVersion36alpha2, serverVersion)
+
+	// service catalog requires the TSB, which requires 3.7
+	serverVersion, _ := c.OpenShiftHelper().ServerPrereleaseVersion()
+	if serverVersion.LT(openshiftVersion37) {
```

Comment:
it does.  After much discussion between @deads2k and I, this is the path of least resistance.  It's not ideal but we're not going to lose too much sleep over it given the tech preview nature of the SC in 3.6.

If you're using the latest(3.7) oc client binary and you want to use the SC/TSB with cluster up, you will need to also use the latest (3.7) images.